Sources/Kitura/bodyParser/BodyParser.swift

```diff
@@ -21,13 +21,20 @@ import Foundation
 
 // MARK: BodyParser
 
-/// The `BodyParser` parses the body of the request prior to sending it to the handler. It reads the Content-Type of the message header and populates the `RouterRequest` body field with a `ParsedBody` enumeration (e.g. json, raw, text, urlEncoded).
+/// The `BodyParser` parses the body of the request prior to sending it to the handler. It reads the Content-Type of the message header and populates the `RouterRequest` body field with a corresponding `ParsedBody` enumeration ("application/json" -> JSON, "text/*" -> text, "application/x-www-form-urlencoded" -> URLEncoded, "multipart/*" -> multiPart, no declared content type -> nil, any other Content-Type -> raw).
```

Comment:
This is not at all easy to read in this format.

I've spoken with David Jones and we suggest removing the following wording:
("application/json" -> JSON, "text/*" -> text, "application/x-www-form-urlencoded" -> URLEncoded, "multipart/*" -> multiPart, no declared content type -> nil, any other Content-Type -> raw).

And instead add a new heading of the same font size as Usage Example but called:
ParsedBody Enumeration
This can then describe the ParsedBody enumeration for each case
e.g. "application/json" to JSON

The last two entries should be described as sentences outside the table (I think). e.g. If the message header has no declared content type the ParsedBody enumeration will contain `nil`. If the message header contains any other content type the ParserdBody enumeration will be.....?